circleci/integration-test

```diff
@@ -93,9 +93,13 @@ download_test_artifacts(){
   for workflowID in $(cat integration-tests.out | jq -r '.testWorkflowIds[]');
   do
     S3_BUCKET_URL="s3://frontend-to-end-tester-files-dev/${workflowID}/artifacts.zip"
+    S3_BUCKET="frontend-to-end-tester-files-dev"
+    S3_ARTIFACTS_KEY="${workflowID}/artifacts.zip"
     # TODO we probably at some point want to separate out the integration test script from the artifacts download
     # that way when we want to use i-t-s for other tests besides fe2e, we can still use this script
-    if ! aws s3 ls $S3_BUCKET_URL >/dev/null 2>&1; then
```

Comment:
What was the problem with this line?

Reply:
the problem was that i didnt include the `--profile` lol. but the reason I changed it is because i realized that if the prefix matches, but not an exact match:
e.g. `bucket1/artifacts.zip` exists in s3 but "aws s3 ls s3://bucket`/artifacts.z", itll still return "truthy".

i dont think this really matters for our use case here, but changed it just in case